packages/sanity/src/core/studio/components/navbar/search/components/filters/documentTypes/DocumentTypesButton.tsx

```diff
@@ -15,7 +15,7 @@ const FALLBACK_PLACEMENTS: Placement[] = ['top-start', 'bottom-start']
 export function DocumentTypesButton() {
   const [open, setOpen] = useState(false)
   const [buttonElement, setButtonElement] = useState<HTMLElement | null>(null)
```

Comment:
Could this also be turned into a ref now?

Reply:
[Not yet](https://github.com/sanity-io/sanity/blob/bedbd5ad561aa7eab0324038dd2f49f708a86d19/packages/sanity/src/core/studio/components/navbar/search/components/filters/documentTypes/DocumentTypesButton.tsx#L39)